doom-modeline.el

```diff
@@ -86,15 +87,15 @@
 
 (doom-modeline-def-modeline 'main
   '(bar workspace-name window-number evil-state god-state ryo-modal xah-fly-keys matches buffer-info remote-host buffer-position parrot selection-info)
-  '(misc-info persp-name lsp irc mu4e github debug fancy-battery minor-modes input-method buffer-encoding major-mode process vcs checker))
+  '(objed-state misc-info persp-name lsp irc mu4e github debug fancy-battery minor-modes input-method buffer-encoding major-mode process vcs checker))
```

Comment:
In my understanding, `objed-state` is kind of modal editing state, like `evil-state` or `god-state`, correct? If I am not wrong, it should be put to left side of mode-line, near `evil-state` and other modal editing indicators.

I've merged all editing state indicators into one `modal` segment in 2140ded3deec9c472081184a6e7ccdb03a5be5f3.

Reply:
Yes, it is a modal indicator, but it doesn't work well in any position to the left of the buffer name.  The indicator looks like `line(w)` or `paragraph(w)` and this changes frequently during editing, causing a big chunk of the modeline to jump around.  The position of `global-mode-string` works well for this indicator.